<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-L22FBCI" name="org.example.testRunner.Runners" tests="19" failures="3" timestamp="2022-07-28T17:27:56 EET" time="232.335" errors="0">
  <testcase name="runScenario" time="10.963" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="13.929" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="24.443" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="9.887" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="9.836" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="14.430" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="11.685" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="11.128" classname="org.example.testRunner.Runners">
    <failure type="java.lang.AssertionError" message="expected [https://demo.nopcommerce.com/iphone-6] but found [https://demo.nopcommerce.com/]">
      <![CDATA[java.lang.AssertionError: expected [https://demo.nopcommerce.com/iphone-6] but found [https://demo.nopcommerce.com/]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at org.example.StepDefs.D06_homeSlidersStepDef.userGoToSecondSliderPage(D06_homeSlidersStepDef.java:47)
at ✽.user go to second Slider page(file:///D:/anan/demonopcommerce/src/main/resources/features/F06_HomeSliders.feature:13)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="11.664" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="11.403" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="13.428" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="11.083" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="12.599" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="11.176" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="10.007" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="11.189" classname="org.example.testRunner.Runners">
    <failure type="java.lang.AssertionError" message="expected [ https://demo.nopcommerce.com/nokia-lumia-1020] but found [https://demo.nopcommerce.com/]">
      <![CDATA[java.lang.AssertionError: expected [ https://demo.nopcommerce.com/nokia-lumia-1020] but found [https://demo.nopcommerce.com/]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at org.example.StepDefs.D06_homeSlidersStepDef.userGoToFirstSliderPage(D06_homeSlidersStepDef.java:29)
at ✽.user go to first Slider page(file:///D:/anan/demonopcommerce/src/main/resources/features/F06_HomeSliders.feature:7)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
  <testcase name="runScenario" time="12.587" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="9.194" classname="org.example.testRunner.Runners"/>
  <system-out/>
  <testcase name="runScenario" time="11.704" classname="org.example.testRunner.Runners">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	this icon doesn&#039;t open new tab expected [2] but found [1],
	expected [https://demo.nopcommerce.com/new-online-store-is-open] but found [https://demo.nopcommerce.com/news/rss/1]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
this icon doesn't open new tab expected [2] but found [1],
expected [https://demo.nopcommerce.com/new-online-store-is-open] but found [https://demo.nopcommerce.com/news/rss/1]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at org.example.StepDefs.D07_followUsStepDef.isOpenedInNewTab(D07_followUsStepDef.java:64)
at ✽."https://demo.nopcommerce.com/new-online-store-is-open" is opened in new tab(file:///D:/anan/demonopcommerce/src/main/resources/features/F07_FollowUs.feature:17)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- org.example.testRunner.Runners -->
